microhapulator/tests/test_mappingqc.py

```diff
@@ -16,51 +16,38 @@
 
 
 def test_mappingqc(tmp_path):
-    marker_counts = data_file("test-mapped-reads.txt")
-    full_refr_counts = data_file("test-fullrefr-mapped-reads.txt")
-    rep_counts = data_file("repetitive-reads.csv")
     figure = tmp_path / "donut.png"
-    obs_data = mhapi.read_mapping_qc(marker_counts, full_refr_counts, rep_counts, figure)
```

Comment:
Some changes to these tests were necessary to account for the use of `samtools stats` output rather than `samtools view` output.